docs/learning-the-basics.md

```diff
@@ -87,15 +87,15 @@ You can customize hot corners through **[System Settings](/docs/learning-the-bas
 ### Workspaces {#workspaces}
 By default, all app windows open on one workspace. However, you can use multiple workspaces to organize your workflow:
 
-* To see an overview of your workspaces, press <kbd>⌘</kbd> + <kbd>S</kbd>
-* To quickly move left or right through your workspaces, press <kbd>⌘</kbd> + <kbd>←</kbd> or <kbd>⌘</kbd> + <kbd>→</kbd>
-* To cycle through your workspaces, press <kbd>⌘</kbd> + <kbd>Tab</kbd>
-* You can jump straight to a specific workspace by pressing <kbd>⌘</kbd> + <kbd>1</kbd> through <kbd>⌘</kbd> + <kbd>9</kbd>. You can always jump to a new workspace with <kbd>⌘</kbd> + <kbd>0</kbd>.
-* You can also drag app windows between workspaces by dragging their icons in the workspace overview, or move them left and right between workspaces using <kbd>⌘</kbd> + <kbd>Alt</kbd> + <kbd>←</kbd> or <kbd>→</kbd> respectively. To see an overview of app windows across all workspaces, press <kbd>⌘</kbd> + <kbd>A</kbd>.
+* To see an overview of your workspaces, press <kbd>Ctrl</kbd> + <kbd>S</kbd>
+* To quickly move left or right through your workspaces, press <kbd>Ctrl</kbd> + <kbd>←</kbd> or <kbd>Ctrl</kbd> + <kbd>→</kbd>
+* To cycle through your workspaces, press <kbd>Ctrl</kbd> + <kbd>Tab</kbd>
+* You can jump straight to a specific workspace by pressing <kbd>Ctrl</kbd> + <kbd>1</kbd> through <kbd>Ctrl</kbd> + <kbd>9</kbd>. You can always jump to a new workspace with <kbd>Ctrl</kbd> + <kbd>0</kbd>.
+* You can also drag app windows between workspaces by dragging their icons in the workspace overview, or move them left and right between workspaces using <kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>←</kbd> or <kbd>→</kbd> respectively. To see an overview of app windows across all workspaces, press <kbd>Ctrl</kbd> + <kbd>A</kbd>.
 
 You can customize these shortcuts through **[System Settings](/docs/learning-the-basics#system-settings)** → **Keyboard** → **Shortcuts** → **Workspaces**.
 
-_Note: <kbd>⌘</kbd> refers to the "super" key. It is also known as the "Windows" key on most PCs or the "Command" key on Macs._
+_Note: <kbd>Ctrl</kbd> refers to the "super" key. It is also known as the "Windows" key on most PCs or the "Command" key on Macs._
```

Comment:
This is not correct, it's still Super, which we represent as ⌘

Reply:
I should have actually read that x)

Reply:
@comradekingu we use the looped square symbol (which is in Unicode) consistently across the OS itself and other documentation. I wrote about the reasoning at https://medium.com/@cassidyjames/why-the-looped-square-symbol-809a3de71bea